magik-squid/src/main/java/nl/ramsolutions/sw/magik/formatting/MagikFormattingSettings.java

```diff
@@ -35,24 +81,44 @@ public char getIndentChar() {
   /**
    * Get the indent width.
    *
-   * @return
+   * @return The indent width, defaults to 8.
    */
   public int getIndentWidth() {
     return this.properties.getPropertyInteger(KEY_MAGIK_FORMATTING_INDENT_WIDTH, 8);
   }
 
+  /**
+   * Check if the formatter should insert a final newline.
+   *
+   * @return True if a final newline should be inserted, defaults to true.
```

Comment:
```suggestion
   * @return True if a final newline should be inserted, defaults to false.
```

Although it differs from https://github.com/StevenLooman/magik-tools/blob/b24b5b985fe4b25ddaa3e576dc5d92cbc65400fe/.vscode/settings.json#L15-L17